### PR TITLE
First Level Derivation feature

### PR DESCRIPTION
**Pull request by Ashish81**
@NicolasDorier  I added Derivativefeature.Direct , now NBXplorer is able to track both 1st level and second level derivations. Please review and suggest for betterment

### CODE REVIEW

**Comment by NicolasDorier** [comment]
Seems good! Can you just fix the indent to tabs?

**Comment by Ashish81** [comment]
It was little difficult Indenting as in VS2017 it shows good, but Github shows incorrect indents.

File: NBXplorer/Repository.cs [hunk 1]
```diff
@@ -589,7 +588,7 @@ private void RefillAvailable(DBreeze.Transactions.Transaction tx, DerivationStra
 					Redeem = derivation.Redeem,
 					DerivationStrategy = strategy,
 					Feature = derivationFeature,
-					KeyPath = DerivationStrategyBase.GetKeyPath(derivationFeature).Derive(index, false)
+					KeyPath = DerivationFeature.Direct == derivationFeature ? new KeyPath(index.ToString()) : DerivationStrategyBase.GetKeyPath(derivationFeature).Derive(index, false)
```
Comment (NicolasDorier) [inline NBXplorer/Repository.cs]: why not just 'DerivationStrategyBase.GetKeyPath(derivationFeature).Derive(index, false)' ?

Comment (Ashish81) [reply NBXplorer/Repository.cs]: Yes, Correct , actually changed the GetKeyPath() function later.
DerivationStrategyBase.GetKeyPath(derivationFeature).Derive(index, false) == new KeyPath().Derive(1)

**Comment by NicolasDorier** [comment]
@Ashish81 I will put an editorconfig for next time so you don't have this problem.

**Pull request merged**